fcrepo-persistence-api/src/main/java/org/fcrepo/persistence/api/common/FileWriteOutcome.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.persistence.api.common;
```

Comment:
Was this not supposed to be a new module `fcrepo-persistence-common` (or `org.fcrepo.persistence.common`) or did I misunderstand the discussion?

Reply:
I had it in `org.fcrepo.persistence.common` originally, but it wasn't available to other modules in that package in fcrepo-persistence-api (probably some maven thing). I can break it out into `fcrepo-persistence-common`, i just wasn't sure if we'd reached agreement on that. If you're okay with it though I can go ahead

Reply:
I'd say break it out into its own module, it's not API stuff and it is common across the persistence layer. So that makes sense to me. Not sure what others think.